presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveBasicTableStatistics.java

```diff
@@ -54,6 +55,32 @@ public void testCreateUnpartitioned()
         }
     }
 
+    @Test
+    public void testCreateExternalUnpartitioned()
+    {
+        String tableName = "test_basic_statistics_external_unpartitioned_presto";
+
+        onPresto().executeQuery(format("DROP TABLE IF EXISTS %s", tableName));
+
+        try {
+            String location = getTableLocation("nation", false);
+            onPresto().executeQuery(format("" +
+                    "CREATE TABLE %s (" +
+                    "   n_nationkey bigint, " +
+                    "   n_regionkey bigint, " +
+                    "   n_name varchar(25), " +
+                    "   n_comment varchar(152)) " +
+                    "WITH (external_location = '%s')",
+                    tableName,
+                    location));
+            BasicStatistics statistics = getBasicStatisticsForTable(onHive(), tableName);
+            assertThatStatisticsAreZero(statistics);
```

Comment:
This is wrong. i checked with `show stats for test_basic_statistics_external_unpartitioned_presto;` that this indeed reports row count 0, but the table is not empty. It should report NULL

Reply:
I guess this is actually a pre-existing thing, is it?